CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 * Python: Added ZDIFFSTORE command ([#1378](https://github.com/aws/glide-for-redis/pull/1378))
 * Python: Added ZDIFF command ([#1401](https://github.com/aws/glide-for-redis/pull/1401))
 * Python: Added BZPOPMIN and BZPOPMAX commands ([#1399](https://github.com/aws/glide-for-redis/pull/1399))
+* Python: Update XADD and XTRIM options (TODO)
```

Comment:
I think it should be in breacking changes section below

Reply:
Will add this PR to line 23, which is new for the release.